lib/unimatrix/authorization/filters/requires_policies.rb

```diff
@@ -80,7 +80,8 @@ def policies
     end
 
     def retrieve_policies( resource_name, access_token, realm )
-      if resource_name && access_token && realm
+      if resource_name && access_token
+        realm    = realm || '*'
```

Comment:
Why did you change this? Endpoints that use this before filter for policies should be scoped to a realm. We need to know which realm they are trying to access to see if they have permissions. A wildcard will find any permissions with that resource in any realm, which is not what we want.